web/src/app/shared/models/project.model.ts

```diff
@@ -7,21 +7,70 @@ import { MonitorModel } from './monitor.model';
 import { PingModel } from './ping.model';
 import { ProjectTokenModel } from './project-token.model';
 
-export class ProjectModel {
-  uid?: string = '';
+/**
+ * Interface for project
+ */
+export class IProject {
+  uid?: string;
   type?: 'private' | 'public' = 'public';
-  title?: string = '';
-  description?: string = '';
+  title?: string;
+  description?: string;
+  url?: string;
+  access?: AccessModel;
+  repositories?: string[];
+  monitors?: MonitorModel[];
+  pings?: PingModel[];
+  tokens?: ProjectTokenModel[];
+  createdOn?: firestore.Timestamp;
+  updatedOn?: firestore.Timestamp;
+}
+
+/**
+ * Project class model
+ */
+export class ProjectModel {
```

Comment:
Oh I just noticed I forgot to implement `IProject` 🤦‍♂

Reply:
I fixed in another PR, plus also other places that used `ProjectModel` and should use `IProject`